torch/_dynamo/config.py

```diff
@@ -322,6 +322,8 @@
 # Skip tracing the torchrec files added to trace_rules.FBCODE_SKIP_DIRS
 skip_torchrec = True
 
+# Don't apply most trace_rules.py rules
+dont_skip_tracing = False
```

Comment:
Does it make sense to name this something more positive?

e.g.
apply_minimal_trace_rules = False for default.
apply_minimal_trace_rules =True is equivalent to dont_skip_tracing=True?

I do get that the decorator is named dont_skip_tracing, which makes sense, so maybe we want consistency

Reply:
I changed my mind, I think dont_skip_tracing is better. It's 1:1 with the decorator.